app/controllers/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
-  before_action :authenticate_user!
+  # before_action :authenticate_user!
```

Comment:
- [ ] Kindly uncomment this line as it is essential to help enforce authentication for certain actions in the controller, i.e., to ensure that only logged-in users are able to access certain pages or perform certain actions